storage/integration_test.go

```diff
@@ -2195,23 +2194,29 @@ func TestIntegration_RequesterPays(t *testing.T) {
 		}
 	}
 	// Object operations.
+	obj1 := uidSpace.New()
```

Comment:
Nit, prefix with a term like `acl-go-test` so it's traceable back to this code section.

Reply:
Done